src/components/ParameterTabsComponent.js

```diff
@@ -102,7 +166,7 @@ export default function ParameterTabsComponent() {
           fetchRiskValue(values);
         }}
         initialValues={userInput}
-        validationSchema={parametersValidationSchema}
+        // validationSchema={parametersValidationSchema}
```

Comment:
Why is the validation removed?

Reply:
Good question, I don't remember but will put it back

Reply:
Found here some bug, yup must be fixed

Reply:
vaylat and route are validated in another function so now validation schema validates only other parameters